### PR TITLE
Update verion of Kabanero to 0.2.1

### LINKED ISSUE #4: error on run appsody deploy
with reference to the tutorial for [springbboo2](https://kabanero.io/guides/collection-springboot2/#deploying-to-kubernetes) encountered the following error on executing ```appsody deploy``` to kuberentes.

using Docker desktop kubernetes

```
[Docker] Step 15/24 : RUN yum upgrade --disableplugin=subscription-manager -y    && yum clean --disableplugin=subscription-manager packages
[Docker]  ---> Running in 8a458ef628cd
[Docker] Red Hat Universal Base Image 8 (RPMs) - AppStre 0.0  B/s |   0  B     00:32
[Docker] Error: Failed to synchronize cache for repo 'ubi-8-appstream'
[Docker] The command '/bin/sh -c yum upgrade --disableplugin=subscription-manager -y    && yum clean --disableplugin=subscription-manager packages' returned a non-zero code: 1
[Error] exit status 1
```

### PR DESCRIPTION
**Pull request by SueChaplain**
Update all instances of Kabanero 0.1.2 to 0.2.1 per issue #4.
Update all references to repository `appsodyhub` to `incubator`
in line with changes at the Appsody project.

Closes: #4

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Review by kilnerm — APPROVED** [review]
LGTM

**Comment by SueChaplain** [comment]
@kilnerm - I had a doubt about:

>Update all references to repository appsodyhub to incubator

Not sure when the latest Appsody CLI is picked up by the project. Any idea?

**Comment by kilnerm** [comment]
@SueChaplain Given the guide states "output similar too" do we really need to be overly concerned on this point?

**Comment by SueChaplain** [comment]
Fair point. @alohr51 - are you able to merge this please? I assume that the changes to the guide will be automatically picked up by the build process?

File: README.adoc [hunk 1]
```diff
@@ -69,25 +69,25 @@ You see an output similar to the following example:
...
 NAME        URL
-*appsodyhub https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
+*incubator https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml
 ----
 
 Next, run the following command to add the Kabanero index:
 [role="command"]
 ----
-appsody repo add kabanero https://github.com/kabanero-io/collections/releases/download/v0.1.2/kabanero-index.yaml
+appsody repo add kabanero https://github.com/kabanero-io/collections/releases/download/v0.2.1/kabanero-index.yaml
```
Comment (alohr51) [inline README.adoc]: this URL returns a 404 (there is no `v` in front of 0.2.0

I would suggest just making this general and point to the latest Collections release such as:
`appsody repo add kabanero-latest https://github.com/kabanero-io/collections/releases/latest/download/kabanero-index.yaml`

I am unsure someone would want to do this (does it pull the latest every time, or is it just set at whatever is the latest at the time of repo add)

if we can do this, you won't have to keep updating this guide every release

**Review by alohr51 — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (SueChaplain) [reply README.adoc]: @kilnerm - what do you recommend? ^^

Comment (kilnerm) [reply README.adoc]: @SueChaplain I don't see a problem with updating to latest. We should probably add a checkpoint before each release to ensure the guide content is not regressed but other than that it makes sense to have people pick up the latest versions.

Comment (SueChaplain) [reply README.adoc]: @alohr51 - the "v" got removed in an earlier release. I'll update this PR. Note that https://kabanero.io/developer/cli/ has the same error and also needs an update.

**Comment by SueChaplain** [comment]
Closing this PR. New PR created https://github.com/kabanero-io/guide-collection-springboot2/pull/6 that also includes the edits from the Eclipse Microprofile review.

**Pull request closed without merging**